pkg/importer/data-processor.go

```diff
@@ -307,8 +307,8 @@ func (dp *DataProcessor) resize() (ProcessingPhase, error) {
 		if err != nil {
 			return ProcessingPhaseError, err
 		}
-		dp.preallocationApplied = dp.preallocation
 	}
+	dp.preallocationApplied = dp.preallocation
```

Comment:
given the current implementation isn't `preallocationApplied` always true?

Reply:
Only if the DV has the preallocation flag set, the value of dp.preallocation will be true in that case, false otherwise

Reply:
I was mistaken when `preallocationApplied` should always be true.  My understanding of the implementation in this PR is:

1) Regardless of what the user `preallocation` setting the disk will always be preallocated when not "converting" so raw, iso, etc
2) User preallocation settings are respected when "converting" from scratch space. 

I am wondering if case 1 above is accurately reflected in `dp.preallocationApplied`?

Reply:
I am not sure I can know at this point in the code, this gets run for multiple different datasources, some of which may preallocate and some may not. I do know that if the flag was set on the importer pod, then the datasource would respect it, if it was not set, it may or may not have preallocated depending on the datsource.